resources/docs/erp_abrufen_egk-source.adoc

```diff
@@ -0,0 +1,271 @@
+Schlüssel für HMAC: 8b6d075d798f6b9aeda939fd10523996c1d14cdc70b4ed922e3e25c172e6dc9d;
+Prüfziffer: SzAxMDEyNDAyMTE1ODU3MzM5MzRVQTFIEyqeauOYbwMbBNdfsTGa/oaJLUv41Bg= (Länge=64=)
+
+[cols="a,a"]
+[%autowidth]
+|===
+|Code   |Type Success
+|200  | OK +
+[small]#Die Anfrage wurde erfolgreich bearbeitet.#
+|Code   |Type Error
+|400  | Bad Request  +
+[small]#Die Anfrage-Nachricht war fehlerhaft aufgebaut.#
+|===
+
+
+=== Fehlerbehandlung
+Ist die Gesundheitskarte gesperrt, bspw. wegen Kartenwechsel oder -verlust, liefert die Operation ReadVSD *keine ReadVSDResponse* sondern einen `SOAP:Fault`. In diesem Fall muss der Anwendungsfall abgebrochen werden.
+
+Bei folgenden Fehlercodes liegt eine gesperrte eGK vor, möglicherweise ist der Patient bereits im Besitz einer neuen eGK.
+
+* Fehlercode 106
+* Fehlercode 107
+* Fehlercode 114
+
+
+== Liste einlösbarer E-Rezepte vom Fachdienst abrufen
+Mit der folgenden Fachdienst-Operation ruft das AVS alle Tasks zu einlösbaren E-Rezepten vom E-Rezept-Fachdienst ab.
+
+*Request*
+[cols="h,a"]
+[%autowidth]
+|===
+|URI        |https://erp.zentral.erp.splitdns.ti-dienste.de/Task?pnw=H4sIACLx4WMA%2FwHqABX%2FPD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiIHN0YW5kYWxvbmU9InllcyI%2FPjxQTiB4bWxucz0iaHR0cDovL3dzLmdlbWF0aWsuZGUvZmEvdnNkbS9wbncvdjEuMCIgQ0RNX1ZFUlNJT049IjEuMC4wIj48VFM%2BMjAyMjA4MzEwODA3MzY8L1RTPjxFPjI8L0U%2BPFBaPlN6QXhNREV5TkRBeU1URTFPRFUzTXpNNU16UlZRVEZJRXlxZWF1T1lid01iQk5kZnNUR2Evb2FKTFV2NDFCZz08L1BaPjwvUE4%2BfXVkL%2BoAAAA%3D
+
+CAUTION: Der Prüfungsnachweis in `Base64(gzip(xml_PN))`-Darstellung muss vor der Übertragung als URL-Parameter *URL-Safe* codiert werden.
+
+|Method     |GET
+|HTTP Header |
+----
+Authorization: Bearer eyJraWQ.ewogImL2pA10Qql22ddtutrvx4FsDlz.rHQjEmB1lLmpqn9J
+----
+
+NOTE: Im http-Header des äußeren http-Requests an die VAU (POST /VAU) sind die Header `X-erp-user: l` und `X-erp-resource: Task` zu setzen.
+|===
+
+*Response*
+[source,xml]
+----
+include::../examples/ti-dienste/task/response_taskGet_eGK.xml[]
+----
+
+NOTE: Alle gelieferten Tasks haben den Status `<status value="ready" />` und können im Folgenden durch die Apotheke einzeln abgerufen und beliefert werden.
+
+NOTE: Eine Sortierung der Ergebnisse der Liste durch den Fachdienst erfolgt nicht. Ab eine Ergebnismenge von 50 Einträge erfolgt ein paging.
+
+NOTE: Je Rezept müssen die Task-ID `<id value="160.000.166.571.419.21" />` und der AccessCode +
+ `<value value="777bea0e13cc9c42ceec14aec3ddee2263325dc2c6c699db115f58fe423607ea" />` ausgelesen und für den Folgeaufruf der
+ `POST /Task/160.000.166.571.419.21/$accept?ac=777bea0e13cc9c42ceec14aec3ddee2263325dc2c6c699db115f58fe423607ea`-Operation zusammengesetzt werden, siehe link:erp_abrufen.adoc#e-rezept-abrufen[E-Rezept abrufen^].
+
+[cols="a,a"]
+[%autowidth]
+|===
+s|Code   s|Type Success
+|200  | OK +
+[small]#Die Anfrage wurde erfolgreich bearbeitet. Die Response enthält die angefragten Daten.#
+s|Code   s|Type Error
+|403 |Forbidden +
```

Comment:
In https://polarion.int.gematik.de/polarion/redirect/project/Mainline_OPB1/wiki/E-Rezept/gemF_eRp_Autorisierung_Apo?selection=ML-136425 sind verschiedene Gründe für den 403 in den Afos mit speziellen Texten für OperationOutcome  angegeben.
Kannst du diese bitte hier übernehmen

Reply:
Sind nach meiner Ansicht 2 Fälle
* Veralteter PNW
* Prüfung ist fehlgeschlagen
korrekt?